### ISSUE TITLE
Constant y_scale for all Quantize/Dequantize Linear layers after Quantization Aware Training

### ISSUE THREAD

**Issue by israelwei** [onnx/tensorflow-onnx#1438]
**Describe the bug**
I trained in Tensorflow 2.3 a Quantization Aware Training model, where the quantization ops were added in the following way:
I added tf.quantization.quantize_and_dequantize after the activation function in the following pattern:
Conv2D --> BatchNorm --> Activation
The exact snytax, if x is the output of the activation function: 
`x = tf.quantization.quantize_and_dequantize(x, input_min=-64, input_max=64, range_given=False)`
I added range_given=False so the min/max values will be taken from the tensor.
After converting the model to ONNX using tf2onnx, I can see in Netron that all QuantizeLinear / DequantizeLinear nodes have the same scale: approximately 0.5039. For example:

![qat_pattern_small](https://user-images.githubusercontent.com/23454156/113707826-222f3e80-96e9-11eb-8641-49439588fcaa.png)

I attach belowalso a visualization in of the pb graph of the Tensorflow model for a quantize_and_dequantize layer with its parameters:
![qat tf parameters](https://user-images.githubusercontent.com/23454156/113708382-d335d900-96e9-11eb-9488-0ad2ef170d44.PNG)

This repeats for every model I am training with QAT and converting to ONNX.
I can see in the tf2onnx source code of [create_qdq_nodes](https://github.com/onnx/tensorflow-onnx/blob/a991437cdd770479f20688110bd2b44ead63b759/tf2onnx/rewriter/quantization_ops_rewriter.py#L28) that tf2onnx decides on the same min_quantized and max_quantized values. I think according to my arguments it's- [-128, 127]
But for some reason even after multiplying with the min/ max values of the QDQ inputs, the scale is always 0.5039.
Why are the final quantization scale values always the same for all layers? I think this is an unwanted behavior for Quantization Aware Training.

**System information:**

OS Platform and Distribution: Ubuntu 18.04
Tensorflow Version: 2.3
tf2onnx version: issue occurs on versions 1.8.4
Python version: 3.6.9

**Expected behavior**
Different quantization scale values for all QuantizeLinear/Dequantize Linear layers after converting the Tensorflow model to ONNX.

**Additional context**
I can't share the relevant onnx model.

**Comment by TomWildenhain-Microsoft** [comment]
The issue is that the converter is ignoring the range_given attribute and is assuming it is True.
